internal/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ add_subdirectory(Fabulist)
 add_subdirectory(glfw)
 add_subdirectory(glm)
 add_subdirectory(GSL)
+add_subdirectory(stduuid)
```

Comment:
This looks to be out of order alphabetically. Can you correct this or put a comment as to why it is this way?

Reply:
I thought the alphabetical order was just entirely broken after the `zlib-ng` library: there's zlib, libpng, freetype, onetbb and spdlog. At this point i'm unsure where to put the line (considering stduuid sort of depends on GSL).